src/app/pages/things/details/things.details.component.html

```diff
@@ -85,8 +85,10 @@
       </nb-card-body>
     </nb-card>
   </div>
-  
-  <div class="col-md-12">  
-    <ngx-message-table [messages]="messages"></ngx-message-table>
+
+  <div class="col-md-12" *ngIf="experimental">
```

Comment:
This one is tricky. If we set this to experimental, we will use the entire message display. Admittedly, message list is not perfect, i.e. we need pagination to make it work properly. My remark about experimental was about the date picker. But yes, if we want to be strict, than we should move message display into experimental until we solve the issue. However, as with things, channels and twins pagination, i would suggest that only date picker goes to experimental if it's possible and sensible.

Reply:
Pagination is related to the component. Datepicker only change the messages input. If you want to remove the experimental fllag I don't see why it should be added elsewhere.

Reply:
Yes, you are right - pagination is related to the component. My point was, we will loose the entire message display if we set the fields to the experimental. In that case, I would rather keep it in production, even if it does not work fully as intended.

Reply:
Or maybe we can show a raw json list in the production and continue to work on this ngx-message-table. Besides, we should probably rename ngx-message-table to something more meaningful, because it's now more than a table.

Reply:
In this case I'll remove the flag, right? 
For the message-table renaming I propose to talk about in this issue: https://github.com/mainflux/ui/issues/152